package.json

```diff
@@ -21,6 +21,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.18.0",
+    "zustand": "^4.4.6",
```

Comment:
What changed your mind about using redux? I do think zustand is better and less cumbersome for our use case. Now if we wanted to use global state for storing vault collections/ vault items we could do it more easily.

Reply:
I hadn't really decided on redux - I looked at redux, recoil and zustand and zustand just seemed like the simplest option for our use case.

Reply:
Made a change to store the symmetric key in local storage, since page reloads were erasing it.

Reply:
Ohh gotcha. I had gotten the impression we were going with redux because when we talked about a global store we talked about redux the most.  

Interesting, I guess it makes sense that it's just in memory and would need to be persisted somehow. If we go to local storage in the browser would we be able to see the key? That wouldn't be a security problem right?